docs/flashbots-data/mev-inspect-py/overview.md

```diff
@@ -6,15 +6,15 @@ title: Overview
 
 Given a block, mev-inspect finds:
 
-- validator payments (gas + coinbase)
-- tokens transfers and profit
-- swaps and [arbitrages](https://twitter.com/bertcmiller/status/1427632028263059462)
+- Validator payments (gas + coinbase)
+- Tokens transfers and profit
+- Swaps and [arbitrages](https://twitter.com/bertcmiller/status/1427632028263059462)
 - ...and more
 
 All data is stored in Postgres for querying and analysis
 
-To get started, check out the [quick start](/flashbots-data/mev-inspect-py/quick-start)
+To get started, check out the [quick start](/flashbots-data/mev-inspect-py/quick-start.md)
```

Comment:
Why add the ".md"? Do we need to do this for internal links to other docs pages? I don't usually include it...

Reply:
For some reason (this was many months ago) the links without the .md-suffix failed to load for me. I've forgotten in which environment this happened (mobile, github editor, vs-code or elsewhere). So I made the edits as the links with .md did work.

Not able to replicate it now though, so this is a bit unnecessary.

Reply:
Huh. I've never had that issue. Could it have to do with the way you've made two pages -- one with the title and the other with the .md content? Do you think we could update to combine those and then link to the page without the .md postfix?